### ISSUE TITLE
关于内积替换欧式距离的问题

### ISSUE THREAD

**Issue by changhexingchen** [HuangCongQing/RandLA-Net-Enhanced#6]
作者您好：
我看内积替换欧式距离的问代码是把空间距离换成了空间距离的平方了。
relative_dis = tf.sqrt(tf.reduce_sum(tf.square(relative_xyz), axis=-1, keepdims=True))
换成了：
relative_dis = tf.reduce_sum(tf.square(relative_xyz), axis=-1, keepdims=True)
请问还有其它代码替换吗？假如只有这一处的话，这个算不上用内积替换欧式距离呀。
